### PR TITLE
Fixing weekly tests to run only one day a week and not for PRs

### PR DESCRIPTION
**Pull request by wdhawkins**

### CODE REVIEW

File: .github/workflows/weekly.yaml [hunk 1]
```diff
@@ -2,13 +2,9 @@ name: Weekly Tests
 
 on:
   schedule:
-    - cron: "0 6 * * *"
+    - cron: "0 6 * * 0"
   push:
     branches: [main]
-  workflow_dispatch:
```
Comment (andrsd) [inline .github/workflows/weekly.yaml:8]: Do you want to keep this, so you can trigger the workflow manually?

**Review by ragusa — APPROVED** [review]

File: .github/workflows/weekly.yaml (hunk 1 above)
Comment (wdhawkins) [reply .github/workflows/weekly.yaml:8]: I thought it might be handy for testing/debugging purposes.

Comment (andrsd) [reply .github/workflows/weekly.yaml:8]: The `workflow_dispatch:` line enables the manual trigger , see [doco](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)

**Review by zhardy-lanl — APPROVED** [review]

File: .github/workflows/weekly.yaml (hunk 1 above)
Comment (wdhawkins) [reply .github/workflows/weekly.yaml:8]: Fixed.

**Review by andrsd — APPROVED** [review]

**Pull request merged**